app/assets/stylesheets/dropzone/dropzone.css.scss

```diff
@@ -171,9 +181,9 @@
   top: 50%;
   left: 50%;
 }
-@media all and (-webkit-min-device-pixel-ratio:1.5),(min--moz-device-pixel-ratio:1.5),(-o-min-device-pixel-ratio:1.5/1),(min-device-pixel-ratio:1.5),(min-resolution:138dpi),(min-resolution:1.5dppx) {
+@media all and (-webkit-min-device-pixel-ratio: 1.5), (min--moz-device-pixel-ratio: 1.5), (-o-min-device-pixel-ratio: 1.5/1), (min-device-pixel-ratio: 1.5), (min-resolution: 138dpi), (min-resolution: 1.5dppx) {
   .dropzone .dz-default.dz-message {
-    background-image: image-url("dropzone/spritemap@2x.png");
+    background-image: url("../images/spritemap@2x.png");
```

Comment:
This kind of references will most certainly be broken on production environments. Have you tried this in production? Using `image-url(...)` is the way to go when using Sprockets.